### PR TITLE
ECG student MRI teacher in model zoo

### PR DESCRIPTION
**Pull request by lucidtronix**
This adds the model file for the paper "Deep Learning to Predict Cardiac Magnetic Resonance-Derived Left Ventricular Mass and Hypertrophy from 12-Lead Electrocardiograms".  Also adds the custom loss function the model used and updates ECG TensorMaps to use instance, and removes augmentation from the tensor_from_file which can be handled inside TensorMaps or DataDescriptions going forward.

### CODE REVIEW

File: ml4h/tensormap/ukb/ecg.py [hunk 1]
```diff
@@ -422,166 +396,78 @@ def ecg_rest_section_to_segment(tm, hd5, dependents={}):
...
     'strip', Interpretation.CONTINUOUS, shape=(600, 8), path_prefix='ukb_ecg_rest', channel_map={'lead': 0}, tensor_from_file=_make_ecg_rest(),
-    dependent_map=ecg_median_1lead, normalization={'zero_mean_std1': 1.0},
+    dependent_map=ecg_median_1lead, normalization=ZeroMeanStd1(),
 )
 
-
 ecg_median_1lead_categorical = TensorMap(
     'median',  Interpretation.CATEGORICAL, shape=(600, 32), activation='softmax', tensor_from_file=_make_ecg_rest(),
-    channel_map=_get_lead_cm(32)[0], normalization={'zero_mean_std1': 1.0},
+    channel_map=_get_lead_cm(32)[0],  normalization=ZeroMeanStd1(),
```
Comment (paolodi) [inline ml4h/tensormap/ukb/ecg.py:462]: Nit: extra space

File: ml4h/tensormap/ukb/mri.py [hunk 2]
```diff
@@ -5,10 +5,10 @@
 import h5py
 import numpy as np
 from tensorflow.keras.utils import to_categorical
-import cv2
-import blosc
+#import cv2
+#import blosc
```
Comment (paolodi) [inline ml4h/tensormap/ukb/mri.py:9]: These two modules seem to be called by a few adiposity TMAPs in `mri.py`. Maybe we could add them as requirements instead of commenting out. No need to address this in this PR.

File: ml4h/metrics.py [hunk 3]
```diff
@@ -125,6 +125,14 @@ def y_true_squared_times_logcosh(y_true, y_pred):
     return K.maximum(1.0+y_true, 1.0)*K.maximum(1.0+y_true, 1.0)*logcosh(y_true, y_pred)
 
 
+def asymmetric_outlier_mse(y_true, y_pred):
```
Comment (paolodi) [inline ml4h/metrics.py:128]: Interesting!

**Review by paolodi — APPROVED** [review]
Great work! I was able to access the model from the zoo and everything looks good to me. I added a couple of inline comments. Please feel free to merge without circling back!

File: ml4h/tensormap/ukb/ecg.py (hunk 1 above)
Comment (lucidtronix) [reply ml4h/tensormap/ukb/ecg.py]: Fixed

File: ml4h/tensormap/ukb/mri.py (hunk 2 above)
Comment (lucidtronix) [reply ml4h/tensormap/ukb/mri.py]: Oops good catch! I uncommented and added a TODO.

File: ml4h/metrics.py (hunk 3 above)
Comment (lucidtronix) [reply ml4h/metrics.py:128]: :)
